mate-session-wayland.pot

```diff
@@ -0,0 +1,31 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR MATE Desktop Environment team
+# This file is distributed under the same license as the mate-session-wayland package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: mate-session-wayland 1.27.0\n"
+"Report-Msgid-Bugs-To: https://github.com/mate-desktop/mate-session-wayland/"
```

Comment:
As you can see here the naming is wrong of the repository in this generated file.
This comes from AC_INIT in configure.
So naming needs updated.